aibolit/patterns/empty_rethrow/empty_rethrow.py

```diff
@@ -19,29 +19,28 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-
-import javalang
-
+from aibolit.ast_framework import ASTNodeType, AST
 from aibolit.utils.ast_builder import build_ast
+from typing import List, Set
 
 
 class EmptyRethrow:
+    '''
+    Check if we throw the same exception as it was caught
+    '''
 
-    def __init__(self):
-        pass
+    def value(self, filename) -> List[int]:
+        total_code_lines: Set[int] = set()
+        ast = AST.build_from_javalang(build_ast(filename))
+        for try_node in ast.get_proxy_nodes(ASTNodeType.TRY_STATEMENT):
+            for throw_node in ast.get_proxy_nodes(ASTNodeType.THROW_STATEMENT):
+                field_catche = try_node.catches
+                if field_catche:
+                    catch_classes = [x.parameter.name for x in field_catche]
```

Comment:
Another misleading name. May be `exceptions_catching_names`.

Reply:
done